pkg/collector/corechecks/servicediscovery/impl_linux.go

```diff
@@ -87,17 +86,34 @@ func (li *linuxImpl) DiscoverServices() (*discoveredServices, error) {
 		}
 	}
 
-	ports, err := li.portPoller.OpenPorts()
+	sysProbe, err := li.getSysProbeClient()
+	if err != nil {
+		return nil, errWithCode{
+			err:  err,
+			code: errorCodeSystemProbeConn,
+			svc:  nil,
+		}
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	ports, err := sysProbe.GetDiscoveryOpenPorts(ctx)
```

Comment:
The timeout seems to short.
I think that should be configurable and longer
What is being done in the network check?